test/modules/TestResultCoordinator/Reports/TestReportMetadataBase.cs

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace TestResultCoordinator.Reports
+{
+    public abstract class TestReportMetadataBase
+    {
+        public TestReportMetadataBase(string testDescription)
+        {
+            this.TestDescription = testDescription;
+        }
+
+        public string TestDescription { get; }
+
+        public override string ToString()
+        {
+            return $"TestDescription: {this.TestDescription}";
```

Comment:
any usage of this method? I saw you implement ToString() for derived test report metadata, or you should call base.ToString method in child class ToString method?

Reply:
Yes, I will call the base class ToString() in the child class